lib/index.js

```diff
@@ -1,10 +1,10 @@
-var Request = require('./message_request'),
-    Response = require('./message_response'),
-    Batch = require('./batch_message'),
-    RadarMessage = function() {};
+var Request = require('./message_request')
```

Comment:
assuming we use Node 10, let's use ES6 standard for Node.JS then.

These `var` should be `const`.